build.fsx

```diff
@@ -132,9 +137,59 @@ Target "RunTests" <| fun _ ->
     let xunitToolPath = findToolInSubPath "xunit.console.exe" "src/packages/xunit.runner.console*/tools"
     printfn "Using XUnit runner: %s" xunitToolPath
     xUnit2
-        (fun p -> { p with OutputDir = testOutput; ToolPath = xunitToolPath })
+        (fun p -> { p with HtmlOutputPath = Some(testOutput @@ "xunit.html") })
         xunitTestAssemblies
 
+Target "StartDbContainer" <| fun _ ->
+    let dockerImage = getBuildParamOrDefault "dockerImage" @"microsoft/mssql-server-windows-express"
```

Comment:
Yes these are resolved, I was seeing errors because I was testing the Powershell script parameter using the format `dockerImage=`, which is how the parameter is used when calling `build.fsx`.  In Powershell, it'd be `-dockerImage <image>` and both work fine, with tags.